tests/test_file.py

```diff
@@ -382,6 +382,57 @@ def _test_file_get_latest(self, replica: Replica):
                 return
         self.fail(f"Failed after {FILE_GET_RETRY_COUNT} retries.")
 
+    def test_file_get_direct(self):
+        self._test_file_get_direct(Replica.aws)
+        self._test_file_get_direct(Replica.gcp)
+
+    def _test_file_get_direct(self, replica: Replica):
+        """
+        Verify that the direct URL option works for GET/ file
+        """
+        file_uuid = "ce55fd51-7833-469b-be0b-5da88ebebfcd"
+        handle = Config.get_blobstore_handle(replica)
+
+        direct_url_req = str(UrlBuilder()
+                             .set(path="/v1/files/" + file_uuid)
+                             .add_query("replica", replica.name)
+                             .add_query("directurl", "True"))
+        presigned_url_req = str(UrlBuilder()
+                                .set(path="/v1/files/" + file_uuid)
+                                .add_query("replica", replica.name))
+        with override_bucket_config(BucketConfig.TEST_FIXTURE):
+            native_resp_obj = self.assertGetResponse(
+                direct_url_req,
+                requests.codes.found,
+                headers=get_auth_header(),
+                redirect_follow_retries=FILE_GET_RETRY_COUNT,
+                min_retry_interval_header=RETRY_AFTER_INTERVAL,
+                override_retry_interval=1,
+            )
+            resp_obj = self.assertGetResponse(
+                presigned_url_req,
+                requests.codes.found,
+                headers=get_auth_header(),
+                redirect_follow_retries=FILE_GET_RETRY_COUNT,
+                min_retry_interval_header=RETRY_AFTER_INTERVAL,
+                override_retry_interval=1,
+            )
+
+            verify_headers = ['X-DSS-VERSION', 'X-DSS-CREATOR-UID', 'X-DSS-S3-ETAG', 'X-DSS-SHA256',
+                              'X-DSS-SHA1', 'X-DSS-CRC32C']
+            native_headers_verify = {k: v for k, v in native_resp_obj.response.headers.items() if k in verify_headers}
+            presigned_headers_verify = {k: v for k, v in resp_obj.response.headers.items() if k in verify_headers}
+            self.assertDictEqual(native_headers_verify, presigned_headers_verify)
+
+            self.assertTrue(
+                native_resp_obj.response.headers['Location'].split('//')[0].startswith(replica.storage_schema))
```

Comment:
nit: If you split on "://" then you could just check that `replica.storage_schema` equals the 0th chunk, instead of using startswith.